src/mpi/romio/test/coll_perf.c

```diff
@@ -63,12 +64,12 @@ int main(int argc, char **argv)
         len = strlen(*argv);
         filename = (char *) malloc(len + 1);
         strcpy(filename, *argv);
-        MPI_Bcast(&len, 1, MPI_INT, 0, MPI_COMM_WORLD);
-        MPI_Bcast(filename, len + 1, MPI_CHAR, 0, MPI_COMM_WORLD);
+        MPI_Bcast(&len, 1, MPI_AINT, 0, MPI_COMM_WORLD);
+        MPI_Bcast_c(filename, len + 1, MPI_CHAR, 0, MPI_COMM_WORLD);
```

Comment:
This means that we can't run the test on MPIs that does not support large count yet. Is this change necessary?

Reply:
no, not necessary... Instead of promoting `len` I can just cast to int -- no one is making a 2 GiB long filename

Reply:
I see. How about `assert` before cast?